doc/_utils/redirects.py

```diff
@@ -0,0 +1,170 @@
+"""
+Copyright (c) 2022 Nordic Semiconductor
+SPDX-License-Identifier: Apache-2.0
+
+This module contains per-docset variables with a list of tuples
+(old_url, new_url) for pages that need a redirect. This list allows redirecting
+old URLs (caused by, e.g., reorganizing doc directories)
+
+Notes:
+    - Please keep lists sorted alphabetically.
+    - URLs must be relative to document root (with NO leading slash), and
+      without the html extension).
+
+Examples:
+
+    ("old/README", "new/index")
+"""
+
+NRF = [
+    ("app_boards", "app_dev/board_support/index"),
+    ("ug_bootloader", "app_dev/bootloaders_and_dfu/bootloader"),
+    ("ug_bootloader_adding", "app_dev/bootloaders_and_dfu/bootloader_adding"),
+    ("ug_bootloader_config", "app_dev/bootloaders_and_dfu/bootloader_config"),
+    ("ug_bootloader_external_flash", "app_dev/bootloaders_and_dfu/bootloader_external_flash"),
+    ("ug_bootloader_testing", "app_dev/bootloaders_and_dfu/bootloader_testing"),
+    ("ug_fw_update", "app_dev/bootloaders_and_dfu/fw_update"),
+    ("app_bootloaders", "app_dev/bootloaders_and_dfu/index"),
+    ("ug_logging", "app_dev/logging/index"),
+    ("ug_multi_image", "app_dev/multi_image/index"),
+    ("app_opt", "app_dev/optimizing/index"),
+    ("app_memory", "app_dev/optimizing/memory"),
+    ("app_power_opt", "app_dev/optimizing/power_opt"),
```

Comment:
since we renamed the file 
```suggestion
    ("app_power_opt", "app_dev/optimizing/power"),
```

Reply:
right, thanks :)